Automattic/jetpack
# Donations Block: unify stripe nudge and refactor donations block to use it

Fixes #24176

#### Changes proposed in this Pull Request:

* Unified the Stripe Nudge components into a single component.
* Updated the Donations block to use the single Strip Nudge component.
* Cleaned up the Donations block.
* Cleaned up the Premium content block.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Make sure that your Stripe account is disconnected.
* Create or edit a Post
* Add a Donations, Payments, and Premium Content block.
* You should see the Stripe connect nudge in every block.

#### Testing video:

https://user-images.githubusercontent.com/1989914/165936272-677496ab-1195-42c8-816e-25d4c27fa5e2.mp4
